### PR TITLE
Modify storage handler

### LINKED ISSUE #22: Create new tests for null or empty ISBN or author in Storage Handler getBookQuery
Currently, there are no tests checking for the correct formatting of a string with with null or empty ISBN or author in the database.

### PR DESCRIPTION
**Pull request by linamontes10-zz**
Add new tests to handle empty and null entries for author and ISBN of a book in the database.
Change ISBN to be string instead of long.

### CODE REVIEW

**Review by JosephBushagour — APPROVED** [review]

File: server/src/main/java/com/google/coffeehouse/common/StorageHandler.java [hunk 1]
```diff
@@ -143,12 +145,20 @@ public static String getBookQuery(DatabaseClient dbClient, String bookId) {
...
       try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
         while (resultSet.next()) {
-          author = resultSet.getString("author");
-          isbn = resultSet.getLong("isbn");
           title = resultSet.getString("title");
+          if (resultSet.isNull("author") || resultSet.getString("author").isEmpty()) {
+            author = NO_AUTHOR;
+          } else {
+            author = "Author: " + resultSet.getString("author");
+          }
```
Comment (JosephBushagour) [inline server/src/main/java/com/google/coffeehouse/common/StorageHandler.java]: if you wanted, you could change this to:
```
author = (resultSet.isNull("author") || resultSet.getString("author").isEmpty())
    ? NO_AUTHOR
    : "Author: " + resultSet.getString("author");
```

There's really no point in this other than decreased line count however.

Comment (linamontes10-zz) [reply server/src/main/java/com/google/coffeehouse/common/StorageHandler.java]: Thanks for the feedback, I like that change and will implement it.

**Review by jimmycolema — APPROVED** [review]
lgtm

File: server/src/main/java/com/google/coffeehouse/common/StorageHandler.java (hunk 1 above)
Comment (jimmycolema) [reply server/src/main/java/com/google/coffeehouse/common/StorageHandler.java]: Also, ternary operators just look clean!

File: server/src/main/java/com/google/coffeehouse/common/StorageHandler.java [hunk 2]
```diff
@@ -129,7 +131,7 @@ public static String getClubQuery(DatabaseClient dbClient, String clubId) {
   public static String getBookQuery(DatabaseClient dbClient, String bookId) {
```
Comment (KeatonTech) [inline server/src/main/java/com/google/coffeehouse/common/StorageHandler.java:128]: Is the plan to eventually switch this to return an instance of Book? Instead of a string

**Review by KeatonTech — APPROVED** [review]

File: server/src/main/java/com/google/coffeehouse/common/StorageHandler.java (hunk 2 above)
Comment (linamontes10-zz) [reply server/src/main/java/com/google/coffeehouse/common/StorageHandler.java:128]: Yes that is the plan. I'll be doing that in my next PR but Joey needs some fixes in this one so the tests all pass on his end.
